[All hosts](security) Update shared app registration steps (#3784)
OfficeDev/office-js-docs-pr: docs/includes/register-sso-add-in-aad-v2-include.md

    The **Add a scope** pane opens.

> 1. In the **Add a scope** pane, specify the scope's attributes. The following table shows example values for and Outlook add-in requiring the `profile`, `openid`, `Files.ReadWrite`, and `Mail.Read` permissions. Modify the text to match the permissions you are using for your add-in.

Reviewer: ```suggestion
1. In the **Add a scope** pane, specify the scope's attributes. The following table shows example values for and Outlook add-in requiring the `profile`, `openid`, `Files.ReadWrite`, and `Mail.Read` permissions. Modify the text to match the permissions your add-in needs.
```

    :::image type="content" source="../images/azure-portal-request-api-permissions-delegated.png" alt-text="The Request API permissions pane with delegated permissions button.":::

> 1. In the **Select permissions** search box, search for the permissions your add-in needs. For example, for an Outlook add-in, you might use profile, openid, Files.ReadWrite, and Mail.Read.

Reviewer: These permissions were in codeticks earlier.
```suggestion
1. In the **Select permissions** search box, search for the permissions your add-in needs. For example, for an Outlook add-in, you might use `profile`, `openid`, `Files.ReadWrite`, and `Mail.Read`.
```
